orm/encoding/ormfield/codec.go

```diff
@@ -78,10 +78,14 @@ func GetCodec(field protoreflect.FieldDescriptor, nonTerminal bool) (Codec, erro
 		} else {
 			return StringCodec{}, nil
 		}
-	case protoreflect.Uint32Kind, protoreflect.Fixed32Kind:
-		return Uint32Codec{}, nil
-	case protoreflect.Uint64Kind, protoreflect.Fixed64Kind:
-		return Uint64Codec{}, nil
+	case protoreflect.Uint32Kind:
+		return CompactUint32Codec{}, nil
+	case protoreflect.Fixed32Kind:
+		return FixedUint32Codec{}, nil
+	case protoreflect.Uint64Kind:
+		return CompactUint64Codec{}, nil
```

Comment:
BTW, do you see Compact integers being used in values? Or we should probably just stick with Protobuf for those? (there's a beginning of slippery slope of wanting to "improve" protobuf)

Reply:
No the protobuf types should stay as they are. Varint encoding is a pretty good encoding if you don't care about database ordering